app/api/hateoas/HateoasFactory.scala

```diff
@@ -28,13 +28,13 @@ class HateoasFactory @Inject() (appConfig: AppConfig) {
   def wrap[A, D <: HateoasData](payload: A, data: D)(implicit lf: HateoasLinksFactory[A, D]): HateoasWrapper[A] = {
     val links = lf.links(appConfig, data)
 
-    api.models.hateoas.HateoasWrapper(payload, links)
+    hateoas.HateoasWrapper(payload, links)
   }
 
   def wrapList[A[_]: Functor, I, D](payload: A[I], data: D)(implicit lf: HateoasListLinksFactory[A, I, D]): HateoasWrapper[A[HateoasWrapper[I]]] = {
-    val hateoasList = payload.map(i => api.models.hateoas.HateoasWrapper(i, lf.itemLinks(appConfig, data, i)))
+    val hateoasList = payload.map(i => hateoas.HateoasWrapper(i, lf.itemLinks(appConfig, data, i)))
 
-    api.models.hateoas.HateoasWrapper(hateoasList, lf.links(appConfig, data))
+    hateoas.HateoasWrapper(hateoasList, lf.links(appConfig, data))
```

Comment:
```suggestion
    val hateoasList = payload.map(i => HateoasWrapper(i, lf.itemLinks(appConfig, data, i)))
   HateoasWrapper(hateoasList, lf.links(appConfig, data))
```

then can delete line 19